src/instructlab/train/linux_train.py

```diff
@@ -173,6 +175,50 @@ def linux_train(
         hpu.init()
         report_hpu_device(device)
 
+    # device register a module, e.g. torch.cpu or torch.cuda
+    device_module = getattr(torch, device.type, None)
+    # bfloat16 is not supported on older CUDA versions and devices
+    # with CUDA support level < 8.0.
+    if hasattr(device_module, "is_bf16_supported"):
+        use_bf16 = device_module.is_bf16_supported()
+        use_fp16 = not use_bf16
+    elif device.type == "cpu":
+        # TODO: check if Torch and CPU support AVX2, F16C, AVX512
+        use_bf16 = False
+        use_fp16 = False
+    else:
+        # assume bf16 supported unless device says otherwise
+        use_bf16 = True
+        use_fp16 = False
+
+    torch_dtype = "auto" if device.type == "cuda" else None
+    if device.type == "cpu":
+        total_memory = psutil.virtual_memory().total / (1024**3)
+        if total_memory < 60:
```

Comment:
```suggestion
        if total_memory < 62:
```

A system with 64GB of RAM, will report:

```
>>> import psutil
>>> mem = psutil.virtual_memory()
>>> mem
svmem(total=67228049408, available=31099351040, percent=53.7, used=35383861248, free=468701184, active=27983499264, inactive=37159084032, buffers=1079336960, cached=30296150016, shared=2109440, slab=1340628992)
```

And we have. `67228049408` Bytes converted to GiB gives us `67228049408 / 1024 ** 3` gives us `62.6 GiB`